sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -586,9 +586,9 @@ async def test_async_queue_by_servicebus_client_receive_batch_with_deadletter(se
 
             async with sb_client.get_queue_receiver(
                     servicebus_queue.name,
-                    sub_queue = SubQueue.DeadLetter,
+                    sub_queue = ServiceBusSubQueue.DeadLetter.value,
```

Comment:
Why the `.value` here? Is this to test it using strings?
Might be good to have some dedicate tests for not using enums. That tests things like case sensitivity etc.

Reply:
Correct, it is to test using strings.  Truthfully I didn't go too much to edge cases because we were relying on the built-in enum string parsing functionality, but it may be more defensive.  I can slip something in there.

(Making sure we're on the same page though, due to the nature of the parse strategy, it is _not_ case insensitive.  Just making sure that was your expectation as well.)

Reply:
Yes that's my expectation. Though in the autorest generated SDKs we now have fancy case-insensitive enums - so we can adopt that as a future extension :)

Reply:
Dedicated tests added++